.pre-commit-config.yaml

```diff
@@ -18,6 +18,7 @@ repos:
     rev: "v5.0.0"
     hooks:
       - id: check-added-large-files
+        exclude: 'data-raw/.*\.rds'
```

Comment:
Adding ^ and $ anchors means it will specifically match files in the data-raw directory that end in rds. This should allow .rds files in the data-raw directory to bypass the check-added-large-files hook without affecting other files.